plugin.d.ts

```diff
@@ -0,0 +1,118 @@
+/// <reference types='node' />
+
+import { FastifyPluginCallback } from 'fastify';
+
+declare module 'fastify' {
+  interface FastifyInstance {
+    cache: AbstractCacheCompliantObject;
+    cacheSegment: string;
+    // etagMaxLife: number;
```

Comment:
Related piece of code :
https://github.com/fastify/fastify-caching/blob/b26b9deeab9775e5edcb759ef1a76ecd1c230491/plugin.js#L85

Didn't find any references in documentation.
Currently this option is only referenced here.

WDYT ? should I add it like this or should I simply remove it ?